forms.html

```diff
@@ -0,0 +1,198 @@
+            <label for="pwd">Preferred Contact Method:</label>
+            <input type="checkbox" id="contact-option1" name="contact-option1" value="Email">
+            <label for="contact-option1">Email</label>
+            <input type="checkbox" id="contact-option2" name="contact-option2" value="Phone">
+            <label for="contact-option2">Phone</label>
+            <input type="checkbox" id="contact-option3" name="contact-option3" value="Postal Mail">
+            <label for="contact-option3">Postal Mail</label>
+        </div>
+        <div class="form-row">
+            <label for="favcolor">Favorite color:</label>
+            <input type="color" id="favcolor" name="favcolor" value="#00e600">
+        </div>
+    </fieldset>
+    <br><br>
+    <fieldset>
+        <legend>Additional Details</legend>
+        <div class="form-row">
+            <label for="source">How did you hear about us?:</label>
+            <select name="source" id="source">
+                <option value="radio">Advertisement on Radio</option>
+                <option value="tv">Advertisement on TV</option>
+                <option value="internet">Internet</option>
+                <option value="friends">Friends</option>
+                <option value="other">Other</option>
+            </select>
+        </div>
+        <div class="form-row">
+            <label for="feedback">Feedback/Comments:</label>
+            <textarea id="feedback" rows="2" cols="26"></textarea>
+        </div>
+    </fieldset>
+    <br><br>
+    <fieldset>
+        <legend>Profile</legend>
+        <div class="form-row">
+            <label for="myfile">Upload a Profile Picture:</label>
+            <input type="file" id="myfile" name="myfile" accept=".jpg, .jpeg, .png">
+        </div>
+        <div class="form-row">
+            <label for="homepage">Add the website link:</label>
+            <input type="url" id="homepage" name="homepage">
+        </div>
+            <div class="form-row">
+                <label for="myRange">Experience level with HTML:</label>
+            </div>
+                <div class="slidecontainer">
+                    <input type="range" min="1" max="10" value="7" class="slider" id="myRange">
+                    <p>Value: <span id="demo"></span></p>
+                </div>
+                <script>
+                    var slider = document.getElementById("myRange");
+                    var output = document.getElementById("demo");
+                    output.innerHTML = slider.value;
+
+                    slider.oninput = function () {
+                        output.innerHTML = this.value;
+                    }
+                </script>
+      </fieldset>
+    <br>
+    <form id="myForm">
+        <input type="submit" value="Submit" style="display: inline-block; margin-right: 15px;">
+        <input type="button" onclick="myFunction()" value="Reset" style="display: inline-block;">
+    </form>
+    <script>
+        function myFunction() {
```

Comment:
Try to always give descriptive names to functions (we will talk about it on JS lessons, but since you already started using it you might be aware of it)

The function name should speak for itself and should give us a clue what does this function do. 

So in your case instead a generic "myFunction" would be better to call it "submit" or "submitFunction" or something similar

Reply:
Thanks for your advice!